### ISSUE TITLE
Dicey Dungeons crashes whenever I try to enter the lady luck fight

### ISSUE THREAD

**Issue by DefaultDaniels** [TerryCavanagh/diceydungeons.com#1770]
Whenever I enter the last hatch the game crashes before I even see anything. The last thing I hear is the sound, which indicates leaving a level. At first I thought it'd work if I just disabled backstage reunion but the problem persisted. Then I tried to re-beat it with jester instead of thief. Same thing. I even tried not getting buster in my party but that wasn't it either. No matter what I do the game closes and  this is the only thing I get
![34](https://user-images.githubusercontent.com/66996847/84754597-a9cc8500-afd1-11ea-8b97-94b463dbabc1.png)

**Comment by TerryCavanagh** [comment]
Hey @DefaultDaniels! This bug was happening with v1.8 of the game, but was (supposedly) fixed in v1.8.1 - what version are you on? If it's v1.8, then updating and restarting should fix it!

**Comment by DefaultDaniels** [comment]
I'm using a 1.8.1 version, however, I must admit I downloaded the game off the internet, since I didn't think it was that good at the time. I ended up beating most episodes in a few days, despite being skeptical at first. I'm sure this game is well worth the money spent now so let me actually buy the game. I'll get back to you to let you know if it fixed the issue or not and most of all, thanks for the game. A LOT
